### DISCUSSION TITLE
Simple PWA

### DISCUSSION THREAD

**Discussion by sergiorodenas** [filamentphp/filament#11236] [Ideas]
What do you think guys about having a very-simple PWA option just so we can install our platform on mobile devices?

**Comment by vitalijalbu** [comment]
Inertiajs here can be a very good tool... @danharrin  👀

**Comment by mahmoud-757** [comment]
I would also like to know how to do it for filament as a full app, or in general.

**Comment by KoljaL** [comment]
To "install" a webapp on a device, you just need to add a manifest and some icons.

Than you have to deal with the small screen and the missing notifications. 

But anyway, I would love to have official support (at least a statement) for filament as PWA.

Edit:
Mind this: https://github.com/silviolleite/laravel-pwa/issues/105
